Library/Homebrew/formula.rb

```diff
@@ -1008,7 +1008,7 @@ def caveats
   # rarely, you don't want your library symlinked into the main prefix
   # see gettext.rb for an example
   def keg_only?
-    keg_only_reason && keg_only_reason.valid?
+    !!keg_only_reason && keg_only_reason.valid?
```

Comment:
Currently, only having `keg_only` throws `Error: wrong number of arguments (0 for 1..2)`.